src/App.scss

```diff
@@ -1,3 +1,10 @@
 iframe {
   display: none;
 }
+
+.has-background-success-light {
+  background-color: #3e8ed0 !important;
+  outline: none;
+  border: none;
+}
```

Comment:
Hm, no need to change this class. For blue button you have `is-info` class, for green background - `.has-background-success-light`.